### PR TITLE
zh-cn(fix): about well-known translation

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/symbol/description/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Symbol('desc').description;  // "desc"
 Symbol('').description;      // ""
 Symbol().description;        // undefined
 
-// well-known symbols
+// 内置通用的（well-known）symbol
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/symbol/description/index.md:32]: https://github.com/mdn/translated-content/pull/10338#discussion_r1038910924

内置通用

Comment (Jack-Zhang-1314) [inline files/zh-cn/web/javascript/reference/global_objects/symbol/description/index.md:32]:
```suggestion
// 内置通用（well-known）symbol
```

File: files/zh-cn/web/javascript/reference/global_objects/symbol/index.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ typeof symObj;  // "object"
 - {{jsxref("Symbol.prototype")}}
   - : `symbol` 构造函数的原型。
 
-### 众所周知的 symbols
+### 内置通用的（well-known）symbol
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/javascript/reference/global_objects/symbol/index.md:73]:
```suggestion
### 内置通用（well-known）symbol
```

File: files/zh-cn/web/javascript/reference/global_objects/symbol/tostring/index.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Object(Symbol("foo")).toString() + "bar"
 ```js
 Symbol("desc").toString();   // "Symbol(desc)"
 
-// well-known symbols
+// 内置通用的（well-known）symbol
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/javascript/reference/global_objects/symbol/tostring/index.md:40]:
```suggestion
// 内置通用（well-known）symbol
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**